src/FlixterHeader.css

```diff
@@ -15,26 +15,38 @@
   height: 100px;
 }
 
-.title {
+/* .title {
```

Comment:
Be sure to not commit commented out code.

Reply:
+1

Reply:
Ok got it! bad habit